src/puppetlabs/puppetdb/metrics/core.clj

```diff
@@ -2,10 +2,12 @@
   (:require [metrics.reporters.jmx :refer [reporter]]
             [metrics.core :refer [new-registry]]))
 
-(defn new-metrics [domain]
+(defn new-metrics [domain & {:keys [jmx?] :or {jmx? true}}]
```

Comment:
This makes it possible to write tests that use metrics, but don't fire up (possibly multi-instance or parallel-test conflicting) jmx support.